src/rdkafka_proto.h

```diff
@@ -726,5 +726,31 @@ rd_kafka_pid_bump(const rd_kafka_pid_t old) {
 
 /**@}*/
 
+/**
```

Comment:
Please move these structs to `rdkafka_request.h` after `rd_kafka_topic_partition_field_t` where we'll keep the Produce and Fetch tags structures too